packages/mermaid/src/diagrams/er/svgDraw.spec.ts

```diff
@@ -0,0 +1,102 @@
+import * as prettier from 'prettier';
+import { describe } from 'vitest';
+import { draw } from './erRenderer-unified.js';
+import { Diagram } from '../../Diagram.js';
+import { addDetector } from '../../diagram-api/detectType.js';
+import erDetector from './erDetector.js';
+import { ensureNodeFromSelector, jsdomIt, SVG_NODE_ID } from '../../tests/util.js';
+import { setConfig } from '../../config.js';
+import { getThemeVariables as getThemeVariablesDark } from '../../themes/theme-dark.js';
+import { getThemeVariables as getThemeVariablesLight } from '../../themes/theme-default.js';
+
+const { id, detector, loader } = erDetector;
+
+addDetector(id, detector, loader); // Add ER schemas to Mermaid
+
+const INPUT_DIAGRAM_TEXT = `\
+erDiagram
+    CUSTOMER ||--o{ ORDER: places
+    ORDER ||--|{ ORDER_ITEM: contains
+    PRODUCT ||--o{ ORDER_ITEM: includes
+    CUSTOMER {
+        string id
+        string name
+        string email
+    }
+    ORDER {
+        string id
+        date orderDate
+        string status
+    }
+    PRODUCT {
+        string id
+        string name
+        float price
+    }
+    ORDER_ITEM {
+        int quantity
+        float price
+    }
+`;
+
+/**
+ * ER boxes are currently **always** drawn in handDrawn mode.
+ * To compare SVG outputs, we must use a common seed to filter out the noise.
+ */
+const TEST_SEED = 666;
+const lightThemeVariables = getThemeVariablesLight();
+const darkThemeVariables = getThemeVariablesDark();
+
+describe('ER diagram SVGs', () => {
+  jsdomIt('should be identical in dark and light themes (classic look)', async () => {
+    prepareSvgNode();
+    setConfig({ theme: 'default', themeVariables: lightThemeVariables, handDrawnSeed: TEST_SEED });
```

Comment:
The `handDrawnSeed` should not be necessary anymore since we are not drawing rough shapes in classic mode anymore.

Will remove when this PR is ready to review.